lib/editor/components/timetable/EditableCell.js

```diff
@@ -298,24 +298,10 @@ export default class EditableCell extends Component {
     const focusedNotEditing = this.props.isFocused && !this.state.isEditing
     const edgeDiff = this.props.isFocused ? 0 : 0.5
     const divStyle = {
-      height: '100%',
-      display: 'inline-block',
       paddingTop: `${3 + edgeDiff}px`,
       paddingLeft: `${3 + edgeDiff}px`,
-      UserSelect: 'none',
```

Comment:
`UserSelect` is react css shorthand for vendor-prefixed properties: https://stackoverflow.com/a/32100497/915811

I think the vendor-specific css for `UserSelect` is missing in the class selector you added.

Reply:
Looks like `style.css` already has a nifty class to accomplish this.